numpy/core/numeric.py

```diff
@@ -667,6 +666,8 @@ def require(a, dtype=None, requirements=None):
          'C' in requirements:
         copychar = 'C'
 
+    arr = array(a, dtype=dtype, order=copychar, copy=False, subok=subok)
+
```

Comment:
And above, the character could be removed from the property list before going into the `for` loop,. Could also use a `Set` instead of a list to ensure no repetitions, but I suspect that would make little difference in practice..